server/handlers/echoHandler.ts

```diff
@@ -0,0 +1,11 @@
+import {Server} from '../server';
+import { Handler } from './types';
+
+export const echoHandler: Handler<any> = (ws, event) => {
+    console.log('received: %s', JSON.stringify(event));
+    Server.clients.forEach((c) => {
```

Comment:
use `broadcast` instead, or just have it `console.log` and create a broadcast handler